lib/SugarClient/client.js

```diff
@@ -42,8 +45,14 @@ class SugarClient {
     return SugarClient.host;
   }
 
+  async refreshToken() {
+    const token = await auth.checkBearerToken();
+    this.authToken = token;
+  }
+
   primusUrl(baseUrl) {
     if (this.userId && this.authToken) {
+      this.refreshToken()
```

Comment:
Maybe call this from the Panoptes client too, so that the Sugar client syncs tokens with the API client.